src/gic_calculator/GIC_difference.py

```diff
@@ -1,30 +1,33 @@
-def calculate_gic_difference(n1_year, n2_year, principal, interest_rate):
+def calculate_gic_difference(n1_year, n2_year, principal, interest_rate1=None, interest_rate2=None):
```

Comment:
Need to import function `interest_calc`

```suggestion
from gic_calculator.interest_calculator import interest_calc
```